meta-flutter/README.md

```diff
@@ -39,5 +55,33 @@ You need to install libsystemd in the same way with the DRM-GBM backend.
 $ bitbake flutter-drm-eglstream-backend
 ```
 
+## SDK toolchain
```

Comment:
Cross-bulding with Yocto SDK. You need to separate the explain how to build Yocto SDK and cross-building flutter-embedded-linux with it.

Reply:
In addition, you need to separate the explain how to build Yocto SDK and cross-building flutter-embedded-linux with it.